sys/src/libc/port/slice.c

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2016 Google Inc.
+ * Dan Cross <crossd@gmail.com>
+ * See LICENSE for license details.
+ */
+
+#include <u.h>
+#include <libc.h>
+
+void PtrSliceInit(PtrSlice *slice)
```

Comment:
Can you clang-format this to Harvey style? Return value on separate line and all of that jazz...